conformance/google-javalite/src/test/kotlin/com/connectrpc/conformance/javalite/ConformanceTest.kt

```diff
@@ -0,0 +1,704 @@
+// Copyright 2022-2023 The Connect Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.connectrpc.conformance.javalite
+
+import com.connectrpc.Code
+import com.connectrpc.ConnectException
+import com.connectrpc.conformance.BaseConformanceTest
+import com.connectrpc.conformance.ServerType
+import com.connectrpc.conformance.v1.ErrorDetail
+import com.connectrpc.conformance.v1.PayloadType
+import com.connectrpc.conformance.v1.StreamingOutputCallResponse
+import com.connectrpc.conformance.v1.TestServiceClient
+import com.connectrpc.conformance.v1.UnimplementedServiceClient
+import com.connectrpc.conformance.v1.echoStatus
+import com.connectrpc.conformance.v1.errorDetail
+import com.connectrpc.conformance.v1.payload
+import com.connectrpc.conformance.v1.responseParameters
+import com.connectrpc.conformance.v1.simpleRequest
+import com.connectrpc.conformance.v1.streamingInputCallRequest
+import com.connectrpc.conformance.v1.streamingOutputCallRequest
+import com.connectrpc.extensions.GoogleJavaLiteProtobufStrategy
+import com.connectrpc.getOrThrow
+import com.connectrpc.protocols.NetworkProtocol
+import com.google.protobuf.ByteString
+import com.google.protobuf.empty
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.async
+import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.withContext
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.fail
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Parameterized
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+
+@RunWith(Parameterized::class)
+class ConformanceTest(
```

Comment:
I don't like the fact that this is a duplicate of the non-lite test (except for the serialization strategy), but I don't know a better way to share logic with different dependencies.